ZIPFoundation.podspec

```diff
@@ -7,12 +7,12 @@ Pod::Spec.new do |s|
   s.social_media_url = 'http://twitter.com/weichsel'
   s.authors = { 'Thomas Zoechling' => 'thomas@peakstep.com' }
   s.source = { :git => 'https://github.com/weichsel/ZIPFoundation.git', :tag => s.version }
-  s.swift_version = '4.0'
+  s.swift_versions = ['4.0', '5.0']
```

Comment:
As we also have some code paths for `SWIFT_VERSION = 4.2`, we maybe should also include `4.2` here.  
(I assume CocoaPods just passed this value to the compiler.  
As far as I know `SWIFT_VERSION` accepts the following values: `4.0`, `4.2` and `5.0`.

Reply:
Jup, let's add 4.2 as well

Reply:
Thank you!